process/guide.md

```diff
@@ -1648,11 +1648,11 @@ implementation.
   * `meta`: this takes a single mal function argument and returns the
     value of the metadata attribute.
   * `with-meta`: this function takes two arguments. The first argument
-    is a mal function and the second argument is another mal
-    value/type to set as metadata. A copy of the mal function is
+    is a mal value and the second argument is another mal
+    value/type to set as metadata. A copy of the mal type is
```

Comment:
This should really be "value" since it's a copy of the value and not a copy of the value's type.

Reply:
Sorry for the delay in getting back to this, I'll update the PR